src/spirc.rs

```diff
@@ -246,7 +295,9 @@ impl SpircTask {
 
         if frame.get_ident() == self.ident ||
            (frame.get_recipient().len() > 0 && !frame.get_recipient().contains(&self.ident)) {
-            return;
+            if frame.get_typ() == MessageType::kMessageTypeNotify {
```

Comment:
I'd like to use librespot with an Unix socket listener listening for commands generated by another process running on the same board. The other process handles input events like _PlayPause_. I don't want to have to instances of a librespot running, one for streaming and one for remote control, but only one librespot instance being capable of doing both. Note, that the size of a librespot binary is more than 2.1 MB and my device is limited in both flash and RAM memory.